src/components/PartnershipCard/PartnershipCard.graphql

```diff
@@ -3,14 +3,18 @@ fragment PartnershipCard on Partnership {
   createdAt
   types {
     value
+    canEdit
```

Comment:
This fragment should _only_ be used for the `PartnershipCard` and what it needs to display. It should be able to change this at will without affecting anything else.

The details/edit scenes should use a different fragment for the needs of that scene.

Reply:
Understood, removed